pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/cleanup/index.md

```diff
@@ -32,42 +31,23 @@ release "elasticsearch-kubeaddons" uninstalled
 
```

Comment:
I'm not sure what happened here, but the output on line 28 (i.e. just before this line) is missing a line.

 We are directing the customer to uninstall 4 things:  kibana-kubeaddons, elasticsearch-curator-kubeaddons, elasticsearch-kubeaddons, elasticsearchexporter-kubeaddons

But the output section only lists three.   It's missing a line for 'kibana-kubeaddons'.   So it should look like this:

release "kibana-kubeaddons" uninstalled
release "elasticsearchexporter-kubeaddons" uninstalled
release "elasticsearch-curator-kubeaddons" uninstalled
release "elasticsearch-kubeaddons" uninstalled